### ISSUE TITLE
Feature: Some insight would be nice and helpfull to keep people having it running

### ISSUE THREAD

**Issue by fabtho** [DP-3T/dp3t-app-android#29]
I do have a demo version installed on my phone. 

I don't know if the app is working in the sens that I don't see anything going on. 

A log would be very nice to see some action, make it transparent whats going on makes it more trustworthy. Also data should exportable. Or even some gamification. (The less contact the better)

We expect people running this for month without feedback in the best case. (NO infection)

See also here: #3

**Comment by markusb** [comment]
I agree with this. The app as it is currently is extremely boring with no indication at all that it is even working and doing some thing. There should be some statistics about how many bluetooth devices it ha seen, how many other Next Step apps, etc. Maybe even with some timing/history graph, so that you can see that you were close to xxx persons this afternoon, because you went grocery shopping.

While we need absolutely to protect privacy, we also need to provide motivation for people to actually run the app. At the moment the only effect visible is that it drains the battery. If it remains that way I do not see it remaining on many phones a long time.
